### PR TITLE
Add a Connections Listener to BDK

### PR DESCRIPTION
**Pull request by anthony-symphony**
Add a Connections Listener to BDK
Fix some Windows based tests

### CODE REVIEW

**Review by symphony-thibault — APPROVED** [review]
LGTM

File: symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/main/java/com/symphony/bdk/bot/sdk/event/model/ConnectionAcceptedEvent.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*
```
Comment (symphony-youri) [inline symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/main/java/com/symphony/bdk/bot/sdk/event/model/ConnectionAcceptedEvent.java:1]: Please change your default template as to not have those comments :)

**Comment by symphony-youri** [comment]
Whats the use case for this listener? (just to get a bit more context and understand why it is useful, could be in the commit message to retrieve it easily by looking at the history)

**Comment by anthony-symphony** [comment]
Added reasoning and removed the comments. Reasoning is to allow bots that need to communicate externally to have the ability to listen to incoming connection requests and accept them.

**Pull request merged**